action.yml

```diff
@@ -1,10 +1,9 @@
 ---
 name: 'Docker-build-release'
-description: 'Build Docker image and push to repository'
+description: 'Build Docker image and push to GHCR'
 inputs:
-  registry:
-    description: 'Registry to which image will be pushed'
-    default: ghcr.io/uclahs-cds
+  organization:
+    description: 'Organizational host for the image. Defaults to the organization of the calling repository.'
```

Comment:
**question (non-blocking)**: Does not having `default` here cause any issues in cases where the organization isn't provided? Ie. is each input not "required" by default?

Reply:
I read [the docs](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_idrequired) as saying that `required` defaults to false. My [test calling workflows](https://github.com/uclahs-cds/docker-internal-tests/actions/runs/9650819693/workflow) did not supply the `organization` and worked just fine.